### DISCUSSION TITLE
Why has the size of the compiled files in version 0.5.8 more than tripled compared to the previous version 0.5.7?

### DISCUSSION THREAD

**Discussion by 5TDg9** [janhq/jan#4016] [General]
Previous versions of jan are about 350 MB but version 0.5.8 is more than 1 GB big?

**Comment by ghost** [comment]
any news? 0.5.9 have same issue i still on 0.5.6

**Comment by A2va** [comment]
Previously on the first start up of Jan it downloaded the engine (llama.cpp, cuda, vulkan), now it is bundled with the app installer, but the overall size should be the same.
